core/services/job/helpers_test.go

```diff
@@ -1,85 +1,33 @@
-package job_test
+package job
 
-import (
-	"fmt"
-	"testing"
+import "github.com/smartcontractkit/chainlink/core/store/models"
 
-	"github.com/BurntSushi/toml"
-	"github.com/jinzhu/gorm"
-	"github.com/stretchr/testify/require"
-
-	"github.com/smartcontractkit/chainlink/core/internal/cltest"
-	"github.com/smartcontractkit/chainlink/core/services/offchainreporting"
-	"github.com/smartcontractkit/chainlink/core/store/models"
-)
-
-const ocrJobSpecText = `
-type               = "offchainreporting"
-schemaVersion      = 1
-contractAddress    = "%s"
-p2pPeerID          = "%s"
-p2pBootstrapPeers  = [
-    "/dns4/chain.link/tcp/1234/p2p/16Uiu2HAm58SP7UL8zsnpeuwHfytLocaqgnyaYKP8wu7qRdrixLju",
-]
-isBootstrapPeer    = false
-keyBundleID        = "%s"
-monitoringEndpoint = "chain.link:4321"
-transmitterAddress = "%s"
-observationTimeout = "10s"
-blockchainTimeout  = "20s"
-contractConfigTrackerSubscribeInterval = "2m"
-contractConfigTrackerPollInterval = "1m"
-contractConfigConfirmations = 3
-observationSource = """
-    // data source 1
-    ds1          [type=bridge name=voter_turnout];
-    ds1_parse    [type=jsonparse path="one,two"];
-    ds1_multiply [type=multiply times=1.23];
-
-    // data source 2
-    ds2          [type=http method=GET url="https://chain.link/voter_turnout/USA-2020" requestData="{\\"hi\\": \\"hello\\"}"];
-    ds2_parse    [type=jsonparse path="three,four"];
-    ds2_multiply [type=multiply times=4.56];
-
-    ds1 -> ds1_parse -> ds1_multiply -> answer1;
-    ds2 -> ds2_parse -> ds2_multiply -> answer1;
-
-    answer1 [type=median                      index=0];
-    answer2 [type=bridge name=election_winner index=1];
-"""
-`
-
-func makeOCRJobSpec(t *testing.T, db *gorm.DB) (*offchainreporting.OracleSpec, *models.JobSpecV2) {
-	t.Helper()
-
-	peerID := cltest.DefaultP2PPeerID
-	ocrKeyID := cltest.DefaultOCRKeyBundleID
-	jobSpecText := fmt.Sprintf(ocrJobSpecText, cltest.NewAddress().Hex(), peerID.String(), ocrKeyID, cltest.DefaultKey)
+func GetORMAdvisoryLockClassID(oi ORM) int32 {
+	return oi.(*orm).advisoryLockClassID
+}
 
-	var ocrspec offchainreporting.OracleSpec
-	err := toml.Unmarshal([]byte(jobSpecText), &ocrspec)
-	require.NoError(t, err)
+func GetORMClaimedJobs(oi ORM) (claimedJobs []models.JobSpecV2) {
+	o := oi.(*orm)
+	o.claimedJobsMu.RLock()
```

Comment:
Is the ORM becoming just a generic junk drawer for all kinds of sync primitives?

Reply:
Nah it only had the claimedJobsMu.

Also, note this is `job.ORM` not `models.ORM`.

It's tightly scoped.